Resources/Locale/en-US/_NF/job/job-description.ftl

```diff
@@ -1,2 +1,3 @@
 job-description-mercenary = Execute the bidding of anyone- for the right price. Enjoy being unbound from the confines of the law.
 job-description-stc = Expertly de-conflict the space around the station and help the NFSD issue fines for overdocked ships.
+job-description-pilot = Take my love, take my land, take me where I cannot stand. I don't care, I'm still free, you can't take the sky from me.
```

Comment:
What?

Reply:
Firefly series quote. Maybe should've went for different one, but couldn't remember that one.

Reply:
Altered the job description to be less cryptic. Now it reads: "Pilot spaceships from point A to B, outmaneuver pirates and dodge asteroids. You are a leaf on the solar wind, let others marvel at how you soar."